bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,849 @@
+# BMW ConnectedDrive Binding
+
+<img align="right" src="./doc/bmw-connected.png" width="150" height="150"/>
+
+The binding provides a connection between [BMW's ConnectedDrive Portal](https://www.bmw-connecteddrive.com/country-region-select/country-region-selection.html) and openHAB.
+All vehicles connected to an account will be detected by the discovery with the correct type 
+
+* Conventional Fuel Vehicle
+* Plugin-Hybrid Electrical Vehicle 
+* Battery Electric Vehicle with Range Extender
+* Battery Electric Vehicle 
+
+In addition properties are attached with information and services provided by this vehicle.
+The provided data depends on 
+
+1. the [Thing Type](#things) and 
+2. the [Properties](#properties) mentioned in Services
+
+Different channel groups are clustering all informations.
+Check for each group if it's supported for this Vehicle.
+
+Please note **this isn't a real-time binding**. 
+If a door is opened the state isn't transmitted and changed immediately. 
+This isn't a flaw in the binding itself because the state in BMW's own ConnectedDrive App is also updated with some delay. 
+
+## Supported Things
+
+### Bridge
+
+The bridge establishes the connection between BMW's ConnectedDrive Portal and openHAB.
+
+| Name                       | Bridge Type ID | Description                                                |
+|----------------------------|----------------|------------------------------------------------------------|
+| BMW ConnectedDrive Account | account        | Access to BMW ConnectedDrive Portal for a specific user    |
```

Comment:
Same for the others.
```suggestion
| BMW ConnectedDrive Account | `account`        | Access to BMW ConnectedDrive Portal for a specific user    |
```

Reply:
Ok, just for my understanding: Putting all Bridge, Thing and Channel Ids in code fences? Is this a new?

Reply:
The Thing Type UID. It's not mandatory.

Reply:
ok, done